<?xml version="1.0"?>

<launch>
   <arg name="robot_name" default="robot"/>
   <arg name="drone_height" default="0.5"/>
   <arg name="safe_dist" default="3.0"/>

   <param name="robot_name" type="str" value="$(arg robot_name)"/>
   <param name="quad_h" type="double" value="$(arg drone_height)"/>
   <param name="safe_dist" type="double" value="$(arg safe_dist)"/>

   <node name="safe_land_server" pkg="quadrotor_control_system" type="safe_land.py" output="screen"/>
</launch>	
